IDEOorg/steps
# #161304967 show number of completed tasks

#### What does this PR do?
- Add implementation to show the number of completed tasks

#### Description of Task to be completed?
- Modify mapStateToProp function in apps/admin/src/layouts/UserLayout.tsx to return completed tasks
- Add markup to display the number of completed tasks

#### How should this be manually tested?
- Clone the repo and check out the branch `ft-show-completed-tasks-count-161304967`. Follow the instructions in the readme to set up your environment. Once setup is done, you should be able to start the application on `http://localhost:3000`. Interact with the app as a coach to test the functionality.

#### What are the relevant pivotal tracker stories?
#161304967

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/30448913/47441351-e073dd00-d7a7-11e8-8125-bc94321df24a.png)
